### PR TITLE
#153 re-list blinded coinjoin criterion

### PR DESCRIPTION
**Pull request by kristovatlas**
This removes  warning from validate_json.py output.

### CODE REVIEW

**Comment by kristovatlas** [comment]
https://github.com/OpenBitcoinPrivacyProject/wallet-ratings/issues/133

File: obpp3.json [hunk 1]
```diff
@@ -1209,6 +1209,12 @@
                                             "id": "OBPPV3/CR30",
                                             "description": "Number of clicks required by user to route outgoing transactions through an anonymizing network",
                                             "effectiveness": 0
+                                        },
+                                        {
+                                            "numeral": "c",
+                                            "id": "OBPPV3/CR48",
+                                            "description": "Mixing is secure against correlation attacks by the facilitator",
```
Comment (kristovatlas) [inline obpp3.json]: The facilitator of the mix does not have an increased ability to reverse the mix beyond the mix participants.

**Comment by dcousens** [comment]
ACK

**Comment by kristovatlas** [comment]
This now matches the version ACKd.

**Pull request merged**